ssh/server/handler/ssh.go

```diff
@@ -169,7 +170,7 @@ func SSHHandler(tunnel *httptunnel.Tunnel) gliderssh.Handler {
 func connectSSH(ctx context.Context, client gliderssh.Session, sess *session.Session, config *gossh.ClientConfig, api internalclient.Client, opts ConfigOptions) error {
 	connection, reqs, err := sess.NewClientConnWithDeadline(config)
```

Comment:
`NewClientConnWithDeadline` can return many errors related to the connection between the server/client to device/agent, so I believe that “failed to authenticate to device”  is the most generic, but “helper” error's message in this context.